src/darwin/frida-helper-backend-glue.m

```diff
@@ -1325,19 +1326,37 @@ static void frida_darwin_helper_backend_launch_using_sbs (NSString * identifier,
 frida_darwin_helper_backend_is_application_process (guint pid)
 {
   gboolean result = FALSE;
+  gchar path[4 * MAXPATHLEN];
   NSAutoreleasePool * pool;
-  NSString * identifier;
+  NSURL * plist_url;
+  NSDictionary * plist = nil;
+  NSString * identifier = nil;
+
+  if (proc_pidpath (pid, path, sizeof (path)) <= 0)
+    return result;
+
+  if (!g_file_test (path, G_FILE_TEST_EXISTS))
+    return result;
```

Comment:
This check can now be removed, since it's now checking if the program's executable exists.

Reply:
oops, right!